assets/src/components/searchForm.tsx

```diff
@@ -287,6 +285,11 @@ export const SearchForm = ({
               searchText={inputText}
               fallbackOption={autocompleteOption(inputText, onSubmit)}
               onSelectVehicleOption={onSelectVehicleOption}
+              onSelectedLocationId={onSelectedLocationId}
+              onSelectedLocationText={(text) => {
+                setAutocompleteEnabled(false)
```

Comment:
The thing is that selecting a text-only suggestion doesn't actually result in a "selection," it results in setting the search query text, so when I was first working on this you would select a text-only suggestion and it would properly set the query and perform the search but not close out the autocomplete. That said I'm not sure this is the best place to trigger this, it just happened to be convenient in terms of where the `setAutocompleteEnabled` callback was, so I'm open to suggestions.

Reply:
Oh right! sorry I misread this as something else, this is good, thanks for explaining!